packages/ui/src/routes/send/SendPage.tsx

```diff
@@ -40,15 +43,28 @@ const SendPage = () => {
     const fromAssetPage = defaultAsset ?? false
     const currentAccount = useSelectedAccount()
 
+    const addressBook = useAddressBook()
+    const addressBookAccounts = useAddressBookAccounts()
+
     // State
     const [selectedAccount, setSelectedAccount] = useState<AccountResult>()
     const [searchString, setSearchString] = useState<string>("")
     const [warning, setWarning] = useState<string>("")
     const [preSelectedAsset, setPreSelectedAsset] = useState<TokenWithBalance>()
     const [isAddress, setIsAddress] = useState<boolean>(false)
 
+    const [addContact, setAddContact] = useState(false)
+
     const searchInputRef = useRef<HTMLInputElement>(null)
 
+    const displayAddToContacts =
+        isAddress &&
+        searchString.toLowerCase() !== currentAccount.address.toLowerCase() &&
```

Comment:
`currentAccount.address` is already checksummed as stated before so here instead of lowercasing the `searchString` multiple times I'd rather apply the checksum too before the comparison